src/test/java/org/jenkinsci/plugins/buildnamesetter/BuildNameSetterTest.java

```diff
@@ -1,72 +1,156 @@
-package org.jenkinsci.plugins.buildnamesetter;

```

Comment:
I don't know what you did here (line breaks change from Windows/Unix to Unix/Windows), but the diff shouldn't look like this.